config/clusters/2i2c/neurohackademy.values.yaml

```diff
@@ -35,8 +35,9 @@ jupyterhub:
           name: The National Institutes of Health grant 2R25MH112480-06
           url: https://reporter.nih.gov/search/ydTvTwXxk0yd6eGdRznbLQ/project-details/10409452
   singleuser:
+    # Pulls content from https://github.com/neurohackademy/nh2022-curriculum using nbgitpuller then opens up /lab
+    defaultUrl: https://neurohackademy.2i2c.cloud/hub/user-redirect/git-pull?repo=https%3A%2F%2Fgithub.com%2Fneurohackademy%2Fnh2022-curriculum&urlpath=lab%2Ftree%2Fnh2022-curriculum%2F&branch=main
```

Comment:
Not sure if this should be a full formed url or just a redirect path?

Reply:
@GeorgianaElena I think this is the first time anyone is trying this ever! Can you try it out on staging to see how it goes?